scripts/exgdas_enkf_fcst.sh

```diff
@@ -108,7 +108,9 @@ export LEVS=${LEVS_ENKF:-${LEVS:-64}}
 
 # nggps_diag_nml
 export FHOUT=${FHOUT_ENKF:-3}
-
+if [[ $CDUMP == "gfs" ]]; then
```

Comment:
Any new code should comply with shellcheck (always use `{ }`).

Reply:
Done